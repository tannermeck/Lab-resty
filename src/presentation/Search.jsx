import React from 'react';

export const Search = ({ verb, link }) => {
    return (
        <>
            <p>{verb}</p>
            <p>{link}</p>
        </>
    )
}