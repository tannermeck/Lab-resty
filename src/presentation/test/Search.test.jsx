import React from 'react';
import { render } from '@testing-library/react'
import { Search } from '../Search.jsx';

describe('should render a list from local storage', () => {
    it('it should render a list containing method and url', () => {
        const { asFragment } = render(
        <Search verb="POST" link="www.me.com" />);
        expect(asFragment()).toMatchSnapshot()
    })
})