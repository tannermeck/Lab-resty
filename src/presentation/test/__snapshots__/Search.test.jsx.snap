// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a list from local storage it should render a list containing method and url 1`] = `
<DocumentFragment>
  <p>
    POST
  </p>
  <p>
    www.me.com
  </p>
</DocumentFragment>
`;
